lab-justin/test/storage-integration-test.js

```diff
@@ -0,0 +1,57 @@
+'use strict';
+
+const expect = require('chai').expect;
+const request = require('superagent');
+const port = process.env.PORT || 3000;
+const server = require('../server');
+const serverUrl = `http://localhost:${port}`;
+
+describe ('testing for storage-integration', function(){
+  before(function(done){
+    if(!server.isRunning){
+      server.listen(port, function(){
+        done();
+      });
+      return;
+    }
+    done();
+  });
+  after(function(done){
+    if(server.isRunning){
+      server.close(function(){
+        server.close(function(){
+          server.isRunning = false;
+          done();
+        });
+        return;
```

Comment:
This looks a little goofy--can you think of a better way to format this to make it readable?